DEVELOPER_STARTING_GUIDE.md

```diff
@@ -0,0 +1,190 @@
+
+## SOLID Library Concepts
+
+### Web **Components** & CSS **Styles**
+
+We are creating a library that comprises a lean combination of reusable [Web Components](https://developer.mozilla.org/en-US/docs/Web/API/Web_components) and [BEM format](https://css-tricks.com/bem-101/) CSS Classes that adhere to [Union design standards](https://www.figma.com/files/1075429990769806468/project/67503549/Solid-DS-Documentation?fuid=883643809929820461). Each have their own technology and purpose:
+
+- **Styles (aka style-components)**: These are standalone CSS files. They don't provide any logic and are often non-interactive.
+- **Components**: These are Web Components. They are often more complex than Styles, and could feature reactivity, state, multiple slots, properties, and more. They can be used in any framework.
+  - [slots](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/slot) allow for easy element nesting ([Lit: Working with Shadow DOM](https://lit.dev/docs/components/shadow-dom/))
+  - [parts](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/part) allow external manipulation of CSS.
+  - [events](https://lit.dev/docs/components/events/) make critical information available outside of a component.
+
+### Samples & Patterns
+
+Alongside the documentation for Styles and Components, we also present Samples and Patterns. These differ in their scope:
+
+- **Samples**: Tied to a specific component, these show its use in a particular context. They are added as "Sample" stories in the component's documentation.
+- **Patterns**: More advanced, they demonstrate how to combine several components to solve a specific problem. They are added as stories in the "Patterns" section.
+
+## Shoelace
+
+The [Shoelace](https://shoelace.style/) web component library is our working inspiration. Components are copied directly from their [source code](https://github.com/shoelace-style/shoelace) and pasted in the `packages/components/src/_components` folder.
+
+_There may have been updates since the files there were copied to our repo!_ In those instances, copy the updated component folder from the Shoelace repository and use that as a starting point.
+
+Additionally we have taken things like the `FormControlController` directly from Shoelace and are slowly adapting them to our needs. If something is broken, missing, or working in an unexpected manner, there may be things that were missed / misconfigured / not yet addressed in our library.
+
+A good rule of thumb: **when in doubt, refer to Shoelace!**
+
+## Storybook
+
+We present the library in a [Storybook](https://storybook.js.org/) with the following sections:
+
+- **DOCS**: All .mdx documentation files.
+- **COMPONENTS**: Web Components built with Lit JS. Often more complex than Styles, and could feature reactivity, state, multiple slots, properties, and more.
+- **UTILITIES**:
```

Comment:
The description is missing here.. Maybe something like this:
UTILITIES: Services that provide essential functionality.

Reply:
```suggestion
- **UTILITIES**: Web Components built with Lit JS, used as SDS-internal helper components. Technically necessary to build other components and generic enough to be used multiple times. Not officially part of the design system library and no respective component existing in Figma.
```

Reply:
what do you think @yoezlem ?